ncm-metaconfig/src/main/metaconfig/ssh/server_attrs.tt

```diff
@@ -0,0 +1,44 @@
+[% spacelist = ['AcceptEnv', 'AllowGroups', 'AllowUsers', 'AuthenticationMethods', 'AuthorizedKeysFile', 'AuthorizedPrincipalsFile',
+    'DenyGroups', 'DenyUsers',
+    'IPQoS',
+    'PermitListen', 'PermitOpen',
+    'RekeyLimit' ] -%]
+[% commalist = ['Ciphers', 'HostKeyAlgorithms', 'HostbasedAcceptedKeyTypes', 'KexAlgorithms', 'MACs', 'PubkeyAcceptedKeyTypes' ] -%]
+[% multilinelist = ['HostKey', 'ListenAddress', 'Port' ] -%]
+[% booleans = ['AllowAgentForwarding',
```

Comment:
these are not needed if they are true pan booleans. you can test those explicitly in TT using `CCM.is_boolean()` (but not in a case block

Reply:
it's `value.is_boolean`; see default case below